assignments/E2.md

```diff
@@ -97,14 +102,13 @@ that it will create in `/proc`
 
     * The `/proc` file must be accessible at `/proc/kdlp`
 
-    * Update your makefile so it can build/clean,
-    load/unload, and test your kernel module
+    * Create a makefile in your `username/E2/` directory to build/clean and load/unload your module
 
     * The makefile's default and `module` targets must build a kernel object file named `kdlp.ko`
 
-    * The makefile's load and unload targets must load and unload your module respectively
+    * The makefile's `load` and `unload` targets must load and unload your module respectively
 
-    * The makefile's `clean` target must remove both test and module build artifacts and should return success even when no artifacts exist to be deleted
```

Comment:
since we are removing the instructions to delete the test program during main makefile clean, we probably need explicitly add instructions about cleaning up the test program with the test makefile clean target